service/src/main/java/bio/terra/axonserver/app/configuration/BeanConfig.java

```diff
@@ -21,4 +24,14 @@ public ObjectMapper objectMapper() {
         .setDateFormat(new ISO8601DateFormat())
         .setDefaultPropertyInclusion(JsonInclude.Include.NON_ABSENT);
   }
+
+  @Bean
+  @Profile("local")
```

Comment:
consider add some documentation in the readme.

what's this for?

Reply:
Its for allowing a locally running UI to query the server. But good call - added comment / updated README